tests/run-pass/aux_test.rs

```diff
@@ -1,4 +1,5 @@
 // aux-build:dep.rs
+// This ignores the test against rustc, but runs it against miri:
```

Comment:
I don't think there's a good reason that this test should be ignored in normal-rustc mode, it's just that compiletest has some surprising behavior: https://github.com/laumann/compiletest-rs/issues/65